### ISSUE TITLE
Refactoring the website: Master Issue

### ISSUE THREAD

**Issue by Aniket-Mishra** [codezoned/codezoned.com#42]
The current Codezoned website uses too many jQuery libraries and has a lot of redundant code that makes future additions and updates confusing and time consuming. This issue is to clean up the website's code and refactor it and fix bugs.
Changes to be made:
1. Shift the navbar icons to the left to make space for future event banners
2. Update the Partners section and change the animations
3. Content changes
4. Redundant code and library removal

**Comment by sureshmangs** [comment]
Can I work on the issue?

**Comment by Aniket-Mishra** [comment]
> Can I work on the issue?

Of course! I'll be adding more issues, do take a look at them too. Thanks!

**Comment by ionicc** [comment]
@Aniket-Mishra Thank you for making this issue! ❤️
